### ISSUE TITLE
CIEDE2000 algorithm incorrect

### ISSUE THREAD

**Issue by jcafaro10** [ViccAlexander/Chameleon#225]
I think this line is wrong:
    CGFloat RT = -2 * sqrt(pow(MeanCPrime, 7) / (pow(MeanCPrime, 7) + pow(25.0, 7))) * sin((60.0 * M_PI/180)* exp(-1 * pow((MeanCPrime - (275.0 * M_PI/180)) / (25.0 * M_PI/180), 2)));

I think it should be this:

    CGFloat RT = -2 * sqrt(pow(MeanCPrime, 7) / (pow(MeanCPrime, 7) + pow(25.0, 7))) * sin((60.0 * M_PI/180)* exp(-1 * pow((MeanHPrime - (275.0 * M_PI/180)) / (25.0 * M_PI/180), 2)));

Based on https://pdfs.semanticscholar.org/969b/c38ea067dd22a47a44bcb59c23807037c8d8.pdf

**Comment by jcafaro10** [comment]
Actually maybe I'm wrong...it's hard to follow the paper

**Comment by rnitta** [comment]
@jcafaro10 
I think you are right
https://github.com/viccalexander/Chameleon/blob/6dd284bde21ea2e7f9fd89bc36f40df16e16369d/Pod/Classes/Objective-C/UIColor%2BChameleon.m#L1026

the last `MeanCPrime` should be `MeanHPrime`
